src/main/player-save.gd

```diff
@@ -9,6 +9,63 @@ Save data is stored as a series of lines. Each line contains a 4-character strin
 	'scen{...}': Scenario data; how many points they scored, how long they took
```

Comment:
These comments are inaccurate.

Reply:
Removed comments; the new format is less surprising than the old format, and does not warrant documentation.